### PR TITLE
Add django

### PR DESCRIPTION
**Pull request by waghanza**

### CODE REVIEW

**Comment by waghanza** [comment]
Hi, 

@tbrand I think this `PR` (in fact conflict resolution) is polluting **git** _history_. To avoid this, I think using `squash` commit (instead of **default** _merge_ one) **SHOULD** fix this ;-)

Regards,

**Comment by tbrand** [comment]
Thanks! :+1:
Actually I did squash merge from some point. Thanks for the advice.

File: tools/src/benchmarker.cr [hunk 1]
```diff
@@ -58,12 +58,8 @@ LANGS = [
   {lang: "python", targets: [
     {name: "sanic", repo: "channelcat/sanic"},
     {name: "japronto", repo: "squeaky-pl/japronto"},
-    # Sorry, it's still not working
-    # For debugging, try; (it will be failed)
-    # ```
-    # ./bin/benchmarker flast
-    # ```
-    # {name: "flask", repo: "pallets/flask"},
```
Comment (tbrand) [inline tools/src/benchmarker.cr]: flask is not working for me now. Did you try it?
```
./bin/benchmerker flask
```

Comment (waghanza) [reply tools/src/benchmarker.cr]: yes, with `make`

it was done in #108, except if I mess up my **working directory** :wink:

Comment (tbrand) [reply tools/src/benchmarker.cr]: Hmm, after merging https://github.com/tbrand/which_is_the_fastest/pull/108, it's still failing for me. But ok, will try it again. Thanks!

**Comment by tbrand** [comment]
Awesome! :tada:

**Pull request merged**